src/main/java/com/uipath/uipathpackage/Utility.java

```diff
@@ -38,79 +33,15 @@
-        String uipathPackageModule = "Import-Module " + escapePowerShellString(new File(tempDir, "UiPath.Extensions/" + getValue(rb, "UiPath.Extensions.Version") + "/UiPathPackage-Module.psd1").getAbsolutePath()) + " -Force";
-        String uipathPowershellModule = "Import-Module " + escapePowerShellString(new File(tempDir, "UiPath.PowerShell/" + getValue(rb, "UiPath.PowerShell.Version") + "/UiPath.PowerShell.psd1").getAbsolutePath()) + " -Force";
-        return getCommand(robotExecutorModule,uipathPackageModule,uipathPowershellModule);
-    }
-
-    /**
-     * Executes the list of powershell commands
-     *
-     * @param listener TaskListener to log the results to console output
-     * @param commands powershell commands in string
-     * @throws IOException in case it fails to read/write in buffered stream while executing command
-     */
-    void execute(@Nonnull TaskListener listener, String... commands) throws IOException {
-        boolean successStatus = true;
-        String commandChain = getCommand(commands);
-        String wrappedCommandChain = "powershell.exe -ExecutionPolicy Bypass Invoke-Expression " + escapePowerShellString(commandChain);
-        // Executing the command
-        Process powerShellProcess = Runtime.getRuntime().exec(wrappedCommandChain);
-        // Getting the results
-        powerShellProcess.getOutputStream().close();
-        String line;
-        listener.getLogger().println("Standard Output:");
-        BufferedReader stdout = new BufferedReader(new InputStreamReader(powerShellProcess.getInputStream(), StandardCharsets.UTF_8));
-        while ((line = stdout.readLine()) != null) {
-            listener.getLogger().println(line);
-        }
-        stdout.close();
-        BufferedReader stderr = new BufferedReader(new InputStreamReader(powerShellProcess.getErrorStream(), StandardCharsets.UTF_8));
-        StringBuilder error = new StringBuilder();
-        while ((line = stderr.readLine()) != null) {
-            successStatus = false;
-            error.append(line);
-        }
-        stderr.close();
-        listener.getLogger().println("Done");
-        if (!successStatus) {
-            throw new AbortException("Error while executing powershell commands for importing modules and pack/deploy\n" + error);
-        }
-    }
-
-    /**
-     * Validates the param for null or empty check
-     *
-     * @param param Param to validate
-     * @param s     Error Message
-     */
-    void validateParams(String param, String s) {
-        if (param == null || param.trim().isEmpty()){
-            throw new InvalidParameterException(s);
-        }
+    private static boolean isCurrentOSWindows() {
+        return System.getProperty("os.name", "generic").toLowerCase(Locale.ENGLISH).contains("win");
     }
 
-    private String getCommand(String... commands) {
+    public static String getCommand(String... commands) {
```

Comment:
Why is this method static? This is used either within this class itself or at a place where we have a reference to instance of Utility class.